package/MDAnalysis/analysis/msd.py

```diff
@@ -27,7 +27,7 @@
 
 :Authors: Hugo MacDermott-Opeskin
 :Year: 2020
-:Copyright: Lesser GNU Public License v2.1+
+:Copyright: Lesser GNU Public License v2.1
```

Comment:
```suggestion
:Copyright: Lesser GNU Public License v2.1+
```
Please don't change our license terms.